Kitodo/src/main/java/org/kitodo/helper/Helper.java

```diff
@@ -341,11 +341,19 @@ public static String getString(Locale language, String key) {
             }
         }
         try {
-            ResourceBundle messages = commonMessages.get(language);
-            if (messages.containsKey(key)) {
-                return messages.getString(key);
+            if (commonMessages.containsKey(language)) {
+                ResourceBundle messages = commonMessages.get(language);
+                if (messages.containsKey(key)) {
+                    return messages.getString(key);
+                }
+            }
+            if (errorMessages.containsKey(language)) {
+                ResourceBundle messages = errorMessages.get(language);
+                if (messages.containsKey(key)) {
```

Comment:
Can you put this out of the brackets to avoid duplicated code?

Reply:
I have moved it own method.